sheet_to_triples/rdf.py

```diff
@@ -108,7 +108,7 @@ def update_model_terms(model, triples):
 
 
 def update_non_uniques(model, non_uniques):
-    model.get('non_uniques', set()).extend(non_uniques)
+    model.get('non_uniques', set()).update(non_uniques)
```

Comment:
Should be?
```python
model.setdefault('non_uniques', set()).update(non_uniques)
```
Also the other place that does `model.get('non_uniques')` needs updating, see after-landing comments from other PR.

Reply:
This probably explains why multiple linking doesn't actually appear to work.